pdc/apps/component/migrations/0004_auto_20150708_0746.py

```diff
@@ -14,6 +13,7 @@ def create_release_component_types(apps, schema_editor):
         'iso',
         'composite'
     ]
+    ReleaseComponentType = apps.get_model('component', 'ReleaseComponentType')
```

Comment:
On latest database, the migration is probably already applied, so it should not matter. Only with new database this migration can't be applied.

It is not possible to import directly, as that would import the latest model (in this case the one with additional `has_osbs` field). We need the historical version here.